src/Core/Migration/V6_6/Migration1739355802FixVatHandling.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_6;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ *
+ * @codeCoverageIgnore
+ */
+#[Package('framework')]
+class Migration1739355802FixVatHandling extends MigrationStep
+{
+    /**
+     * @return array<string, string>
+     */
+    private const VAT_PATTERNS = [
+        'BE' => 'BE(\d{10}$|^0\d{9})',
```

Comment:
this must be wrong. shouldn't it be `BE\d{10}`?

Reply:
Well technically, no. But just checking 10 numbers is actually better. Because as long as it's 10, 0+9 numbers will pass.